### DISCUSSION TITLE
Madmax on Fedora

### DISCUSSION THREAD

**Discussion by sadreddini** [madMAx43v3r/chia-plotter#865] [General]
Hi,
I am wondering if Madmax can be installed and run on Fedora.
If yes can somebody please list the instructions to install?
Any suggestion is appreciated.

**Comment by ataa** [comment]
I have MadMax on CentOS 7 and 8 machines, it should work on Fedora, follow CentOS installation steps.

**Reply by sadreddini** [reply]
Hi,
Thanks for your reply.
Actually I tried installing instructions for CentOs before posting this discussion but I get this error message:
Command:
sudo yum install epel-release -y
Result:
No match for argument: epel-release
Error: Unable to find match: epel-release

**Reply by ataa** [reply]
> EPEL is a Fedora subproject which builds packages from the Fedora collection on the Red Hat Enterprise Linux platform. If you want these packages and are on RHEL, you need to enable EPEL. If you want them on a Fedora OS (Fedora Server, Fedora Workstation, etc.), they're already available by default.

**Reply by sadreddini** [reply]
I need to mention that I am totally newbie to Linux.
Can you please write down all commands and instruction required to install and run Madmax in Fedora?

**Comment by kybash** [comment]
CentOS is a little outdated, some of those steps aren't needed with fedora, this worked fine here...

`sudo dnf install cmake libsodium libsodium-static`
`git clone https://github.com/madMAx43v3r/chia-plotter.git`
`cd chia-plotter/`
`git submodule update --init`
`./make_devel.sh`
`./build/chia_plot --help`
